src/wwwroot/js/site.js

```diff
@@ -2,3 +2,26 @@
 // for details on configuring this project to bundle and minify static web assets.
 
 // Write your JavaScript code.
+$(function() {
+    $(".comment-btn").click(function(){
+        addComment();
+    });
+});
+
+
+
+function addComment() {
+    $.ajax ({
+        type: 'POST',
+        url: '/api/v1/CommentApi/AddComment/' + $(".comment-btn").data().movieid,
+        data: {
+            comment: $("#userComment").val(),
+        },
+        success: function(response){
+            var ul = document.getElementById("commentList");

```

Comment:
For consistency, i recommend sticking to only one way of refferencing DOM objects. If you've used jQuery objects above, it might be better to use jQuery objects in all the documment (it won't necessary mean a difference in performance, but it makes code more readable)